pkg/tf2pulumi/convert/tf.go

```diff
@@ -2155,6 +2155,18 @@ func (s *scopes) generateUniqueName(name, prefix, suffix string) string {
 	}
 }
 
+func (s *scopes) getOrAddOutput(name string) string {
+	root, has := s.roots[name]
```

Comment:
Your gonna have to split roots up I think. It's not just a map of name but also if it's a resource or datasource.